tests/unit/ember-sinon-qunit-test.js

```diff
@@ -2,6 +2,8 @@ import { module } from 'qunit';
 import { resolve } from 'rsvp';
 import test from 'ember-sinon-qunit/test-support/test';
 import assertSinonInTestContext from '../helpers/assert-sinon-in-test-context';
+import Ember from 'ember';
```

Comment:
I *think* you can do `import debug from '@ember/debug';` instead of pulling in Ember here. Looking through the code, `debug` is exported as part of an object, which is how we're able to use destructuring to import it.

https://github.com/emberjs/ember.js/blob/v3.12.0/packages/%40ember/debug/index.ts#L342

Reply:
Unfortunately not:

```
Build Error (broccoli-persistent-filter:Babel > [Babel: ember-sinon-qunit]) in dummy/tests/unit/ember-sinon-qunit-test.js

/Users/gorner/Development/ember-sinon-qunit/dummy/tests/unit/ember-sinon-qunit-test.js: @ember/debug does not have a default export
```

I'm not surprised as it still compiles down to `Ember.deprecate` when built:

```
define("ember-sinon-qunit/test", ["exports", "ember-qunit", "ember-sinon-qunit/utils/config"], function (_exports, _emberQunit, _config) {
  "use strict";

  Object.defineProperty(_exports, "__esModule", {
    value: true
  });
  _exports.default = test;
  // Global sinon config setup
  (0, _config.commonConfig)();

  function test(testName, callback) {
    (true && !(false) && Ember.deprecate('This pattern is now deprecated. Please import the `setupSinon` method instead.', false, {
      id: 'ember-sinon-qunit.test',
      until: '5.0.0',
      url: 'https://github.com/elwayman02/ember-sinon-qunit#migration-from-older-versions-of-ember-sinon-qunit'
    }));
    (0, _config.wrapTest)(testName, callback, _emberQunit.test);
  }
});
```

Reply:
RIP